df.map.xml

```diff
@@ -85,9 +85,9 @@
 
     <bitfield-type type-name='tile_designation'>
         <flag-bit name='flow_size' count='3' comment='liquid amount'/>
-        <flag-bit name='pile' comment='stockpile?'/>
-        <flag-bit name='dig' count='3' type-name='tile_dig_designation'/>
-        <flag-bit name='smooth' count='2'/>
+        <flag-bit name='pile' comment='stockpile; Adventure: lit'/>
```

Comment:
I confirmed that only `<flag-bit>`s are legal inside a `<bitfield-type>` (other than a few basic common elements like `<comment>`).

One alternative I _might_ consider is defining a separate `tile_designation_advmode` bitfield-type, then using a union of the two types wherever it is used. This is probably not a big change - there are only three places where `tile_designation` is used as a member, and two have "liquid" in the member names. I'm not familiar enough to know why that is. The third one is `map_block::designation` which is a `tile_designation[16][16]`, which I'm used to.

It would be a bit annoying to keep the majority of the unchanged fields in sync between the two types, though. Do you know if any bits are _unused_ in adventure mode, like the traffic bits? If only a small subset of bits are used in adv mode, I'd say a union might make sense. Otherwise, I'm totally fine with the comment approach for now.

Reply:
Traffic bits are saved when retiring a fort, but don't seem to be respected in adventure mode. The rest work exactly as they do in fort mode because they're needed for tiles and fluids.